### ISSUE TITLE
AWS SQS V3 don't send instrumentation data

### ISSUE THREAD

**Issue by netanelep** [open-telemetry/opentelemetry-js-contrib#1599]
### What version of OpenTelemetry are you using?
```
"@opentelemetry/api": "^1.4.1",
"@opentelemetry/instrumentation": "^0.41.0",  
"@opentelemetry/instrumentation-aws-sdk": "^0.35.0",  
"@opentelemetry/resources": "^1.15.0",  
"@opentelemetry/sdk-trace-node": "^1.15.0",  
"@opentelemetry/sdk-trace-web": "^1.15.0",  
"@opentelemetry/semantic-conventions": "^1.15.0"
```

SQS client version `"@aws-sdk/client-sqs": "^3.370.0"`

### What version of Node are you using?
16.14.2

### What did you do?
I installed the `instrumentation-aws-sdk` and the trace id and other tracing data are not sent.

### What did you expect to see?
Tracing data send and retrieve correctly between services using SQS

### What did you see instead?
no data

### Additional context
Here is an code example where there isn't any data on the sqs message

```js
const { NodeTracerProvider  } = require("@opentelemetry/sdk-trace-node");
const { registerInstrumentations } = require("@opentelemetry/instrumentation");
const { trace } = require("@opentelemetry/api");
const {
  AwsInstrumentation,
} = require("@opentelemetry/instrumentation-aws-sdk");
const provider = new NodeTracerProvider();
provider.register();

registerInstrumentations({
  instrumentations: [
    new AwsInstrumentation({
      preRequestHook: (span, message) => {
        console.log('preRequestHook running')
      },
      sqsProcessHook: (span, message) => {
        console.log('sqsProcessHook running')
      }
    }),
  ],
});

require('dotenv').config()
const { SQSClient, SendMessageCommand } = require('@aws-sdk/client-sqs')

const url = '...'
const sqs = new SQSClient({
    region: '...',
    endpoint: '...'
})

async function main() {
    const tracer = trace.getTracer('task-tracer')
    await tracer.startActiveSpan('task', async (span) => {
        const sc = new SendMessageCommand({
            QueueUrl: url,
            MessageBody: '123'
        })
    
        const response = await sqs.send(sc)
        console.log(Object.getOwnPropertyDescriptors(response))

        span.end()
    })
}

main()
    .then(() => console.log('main'))
    .catch(err => console.log(err))
```

Print result
```
{
  '$metadata': {
    value: {
      httpStatusCode: 200,
      requestId: undefined,
      extendedRequestId: undefined,
      cfId: undefined,
      attempts: 1,
      totalRetryDelay: 0
    },
    writable: true,
    enumerable: true,
    configurable: true
  },
  MD5OfMessageBody: {
    value: '...',
    writable: true,
    enumerable: true,
    configurable: true
  },
  MessageId: {
    value: '...',
    writable: true,
    enumerable: true,
    configurable: true
  },
  SequenceNumber: {
    value: '...',
    writable: true,
    enumerable: true,
    configurable: true
  }
}
```

**Comment by Fortiz2305** [comment]
I think this is related to #1596

**Comment by carolabadeer** [comment]
Same issue as #1596

**Comment by dyladan** [comment]
Closed as duplicate of #1596 and #1509

**Comment by netanelep** [comment]
@dyladan @carolabadeer still don't work with "@opentelemetry/instrumentation-aws-sdk": "^0.36.0"

**Comment by carolabadeer** [comment]
There is an ongoing investigation into issues with SQS specifically that is being tracked in #1477 - please post any reproduction examples/questions related to SQS in that issue. 

Otherwise, #1509 was tracking the change in `aws-sdk` v3.363.0 causing the AWS SDK instrumentation as a whole to break, and that has been fixed in #1626. @dyladan we can also resolve #1509 and keep #1477 open for tracking the SQS behavior investigation
